daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -2852,8 +2852,8 @@ abstract class ErrorWarningBase(n: NodeSeq, parent: TestCase) {
 
   protected def diagnosticNodes: Seq[Node]
 
-  lazy val messages = diagnosticNodes.map {
-    _.text
+  lazy val messages: Seq[String] = diagnosticNodes.map {
+    _.text.trim
```

Comment:
Just verifying, trim removes leading and trailing whitespace, but doesn't collapse interior whitespace. Am I correct?